src/sentry/mediators/sentry_apps/updater.py

```diff
@@ -62,8 +62,11 @@ def _update_status(self):
 
     @if_param("scopes")
     def _update_scopes(self):
-        if self.sentry_app.status == SentryAppStatus.PUBLISHED:
-            raise APIError("Cannot update scopes on published App.")
+        if (
+            self.sentry_app.status == SentryAppStatus.PUBLISHED
+            and self.sentry_app.scope_list != self.scopes
+        ):
+            raise APIError("Cannot update scopes on a published integration.")
```

Comment:
I know we call it scopes in the code, but we do present it in the UI as 'permissions' so maybe we should switch to this?